common/app/views/fragments/newHeader.scala.html

```diff
@@ -26,16 +23,17 @@ <h1 class="u-h">The Guardian</h1>
         @fragments.nav.editionPicker()
 
         <nav class="new-header__nav">
-            <a href="#main-menu" class="new-header__nav__menu-button js-change-link" tabindex="0">
-                <i class="new-header__veggie-burger-icon">
-                </i>
+            @NewNavigation.topLevelSections.map { section =>
+                @primaryLinks(section.name)
+            }
+            <label for="main-menu-toggle" class="new-header__nav__menu-button js-change-link" tabindex="0">
+                <i class="new-header__veggie-burger-icon"></i>
                 <span class="u-h">Menu</span>
-            </a>
-            <ul class="new-header__nav__list">
-                @NewNavigation.topLevelSections.map { section =>
-                    @primaryLinks(section.name)
-                }
-            </ul>
+            </label>
+            <input type="checkbox" id="main-menu-toggle" class="new-header__nav__button js-enhance-checkbox" aria-controls="main-menu">
```

Comment:
As per [this comment](https://github.com/guardian/frontend/pull/14356#r79419807), we won't focus on tabbing in this PR.

I don't believe void tags like `<input>` need to be closed in HTML5, but I'll defer to any style rules that say otherwise 😄